src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -1982,7 +1982,7 @@ public int getSize()
          * The height of the Attachment if this Attachment is an image/video.
          * <br>If this Attachment is neither an image, nor a video, this returns -1.
          *
-         * @return int containing image/video Attachment height.
+         * @return int containing image/video Attachment height, or -1 if attachment is neither image nor video.
```

Comment:
This is already explained in the description above anyways but sure

Reply:
I added this as there are methods with a similar design.
For example do `Message#getTimeEdited` and `Message#getMember` mention in both description and `@return` that they may return null even if according to your message now, this would only be required in the description itself.